### PR TITLE
Hide badges on plans page in the start-writing flow

### PR DESCRIPTION
**Pull request by paulopmt1**
Related to 2489-gh-Automattic/dotcom-forge

## Proposed Changes

* Hidden two badges on plans page in the start-writing flow

image

## Testing Instructions

* Access the plans page in the new start-writing flow and ensure the first and last plans have those hidden.

## Pre-merge Checklist

- [x] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: client/landing/stepper/declarative-flow/internals/steps-repository/plans/style.scss [hunk 1]
```diff
@@ -199,4 +199,9 @@
 	#step-header {
 		margin: 0;
 	}
+
+	.plan-features-2023-grid__popular-badge.is-current-plan,
+	.plan-features-2023-grid__popular-badge.is-business-plan {
+		display: none;
+	}
```
Comment (DustyReagan) [inline client/landing/stepper/declarative-flow/internals/steps-repository/plans/style.scss:206]: Should we be more specific and add `.start-writing` as the root element here?

Comment (paulopmt1) [reply client/landing/stepper/declarative-flow/internals/steps-repository/plans/style.scss:206]: GH preview didn't show, but we're inside `.start-writing` here. Is that what you meant?

image

Comment (DustyReagan) [reply client/landing/stepper/declarative-flow/internals/steps-repository/plans/style.scss:206]: Haha! So we are! 😅 LGTM!

**Review by DustyReagan — APPROVED** [review]

**Review by lupus2k — APPROVED** [review]
LGTM 🚀

**Pull request merged**